roachtest: tpccbench/nodes=9/cpu=4/multi-region failed

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1909750&tab=buildLog) on [master@3b612692db93aa7c87493705e1fad85c9c664f6c](https://github.com/cockroachdb/cockroach/commits/3b612692db93aa7c87493705e1fad85c9c664f6c):

```
	cluster.go:2444,tpcc.go:731,tpcc.go:572,test_runner.go:753: monitor failure: unexpected node event: 10: dead
		(1) attached stack trace
		  | main.(*monitor).WaitE
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2432
		  | main.(*monitor).Wait
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2440
		  | main.runTPCCBench
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:731
		  | main.registerTPCCBenchSpec.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:572
		  | main.(*testRunner).runTest.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (2) monitor failure
		Wraps: (3) unexpected node event: 10: dead
		Error types: (1) *withstack.withStack (2) *errutil.withMessage (3) *errors.errorString

	cluster.go:1481,context.go:135,cluster.go:1470,test_runner.go:825: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1909750-1588231526-89-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		1: dead
		3: dead
		2: dead
		8: skipped
		6: 3860
		5: 4528
		7: 4179
		12: skipped
		11: dead
		9: dead
		10: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead, 3: dead, 2: dead, 11: dead, 9: dead, 10: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) 1: dead, 3: dead, 2: dead, 11: dead, 9: dead, 10: dead
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1129
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:272
		  | github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:766
		  | github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:852
		  | github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:800
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1793
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Error types: (1) errors.Unclassified (2) *errors.fundamental

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=1909750&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #46387 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46343 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

```
23:12:15 test.go:190: test status: running benchmark, warehouses=3425
23:12:15 cluster.go:2076: [w0] > ./workload run tpcc --warehouses=5000 --active-warehouses=1141 --tolerate-errors --scatter --ramp=3m0s --duration=15m0s --partitions=3 --partition-affinity=2 {pgurl:9-11} --histograms=perf/warehouses=1141/stats.json
23:12:15 cluster.go:2076: [w0] > ./workload run tpcc --warehouses=5000 --active-warehouses=1141 --tolerate-errors --scatter --ramp=3m0s --duration=15m0s --partitions=3 --partition-affinity=1 {pgurl:5-7} --histograms=perf/warehouses=1141/stats.json
23:12:15 cluster.go:2076: [w0] > ./workload run tpcc --warehouses=5000 --active-warehouses=1141 --tolerate-errors --scatter --ramp=3m0s --duration=15m0s --partitions=3 --partition-affinity=0 {pgurl:1-3} --histograms=perf/warehouses=1141/stats.json
3: 13567
1: 15583
2: 14099
5: 15242
6: 14815
7: 14826
9: 16751
11: 14280
10: 14178
23:30:25 test.go:190: test status: getting perf/warehouses=1141/stats.json
23:30:25 cluster.go:352: > /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod get teamcity-1912287-1588318069-90-n12cpu4-geo:8 perf/warehouses=1141/stats.json /home/agent/temp/buildTmp/roachtest-tpcc087497626/3425.1-stats.json
teamcity-1912287-1588318069-90-n12cpu4-geo: getting (scp) perf/warehouses=1141/stats.json /home/agent/temp/buildTmp/roachtest-tpcc087497626/3425.1-stats.json
.
   8: done
23:30:27 test.go:190: test status: 
23:30:44 test.go:190: test status: getting perf/warehouses=1141/stats.json
23:30:44 cluster.go:352: > /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod get teamcity-1912287-1588318069-90-n12cpu4-geo:4 perf/warehouses=1141/stats.json /home/agent/temp/buildTmp/roachtest-tpcc087497626/3425.0-stats.json
teamcity-1912287-1588318069-90-n12cpu4-geo: getting (scp) perf/warehouses=1141/stats.json /home/agent/temp/buildTmp/roachtest-tpcc087497626/3425.0-stats.json

   4: done
23:30:44 test.go:190: test status: 
00:24:59 test_runner.go:773: [w0] dumped stacks to __stacks
00:24:59 test.go:190: test status: stopping cluster
00:24:59 cluster.go:352: > /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod stop --sig=11 teamcity-1912287-1588318069-90-n12cpu4-geo:1-12
teamcity-1912287-1588318069-90-n12cpu4-geo: stopping.
00:25:01 test.go:325: test failure: 	test_runner.go:800: test timed out (10h0m0s)
```

@nvanbenschoten this looks like we genuinely ran into the timeout.

We spent ~4h to get data into the cluster:

```
14:26:47 cluster.go:2076: [w0] > ./workload fixtures load tpcc --warehouses=5000 --scatter --checks=false --partitions=3 --zones="us-east1-b,us-west1-b,europe-west2-b" {pgurl:1}
18:53:40 tpcc.go:646: waiting 1h40m0s for rebalancing
```

Then we die in the 8th benchmark pass. It's interesting that we have.. quite a bit of variance here:

```
22:12:57 tpcc.go:850: --- PASS: tpcc 3375 resulted in 13318.8 tpmC (93.9% of max tpmC)
22:32:35 tpcc.go:854: --- FAIL: tpcc 3775 resulted in 5336.5 tpmC and failed due to efficiency value of 33.653741956994836 is below passing threshold of 85
```

but maybe that's expected. I feel like you've done something about these tests timing out recently. Is that already reflected here?

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072412&tab=buildLog) on [master@e3fb5aa18d0f5064f7ba5d4df3864e94b3abb96d](https://github.com/cockroachdb/cockroach/commits/e3fb5aa18d0f5064f7ba5d4df3864e94b3abb96d):

```
		  | I200708 15:29:54.286031 47 ccl/workloadccl/fixture.go:586  loaded 641 MiB table new_order in 4m15.072608877s (45000000 rows, 0 index entries, 2.5 MiB)
		  | W200708 16:11:30.232418 1 ccl/workloadccl/fixture.go:497  error enabling automatic stats: dial tcp 10.142.15.193:26257: connect: connection refused
		  | Error: restoring fixture: dial tcp 10.142.15.193:26257: connect: connection refused
		  | Error: COMMAND_PROBLEM: exit status 1
		  | (1) COMMAND_PROBLEM
		  | Wraps: (2) Node 4. Command with error:
		  |   | ```
		  |   | ./workload fixtures load tpcc --warehouses=5000 --scatter --checks=false --partitions=3 --zones="us-east1-b,us-west1-b,europe-west2-b" {pgurl:1}
		  |   | ```
		  | Wraps: (3) exit status 1
		  | Error types: (1) errors.Cmd (2) *hintdetail.withDetail (3) *exec.ExitError
		  |
		  | stdout:
		Wraps: (9) exit status 20
		Error types: (1) *withstack.withStack (2) *errutil.withMessage (3) *withstack.withStack (4) *errutil.withMessage (5) *withstack.withStack (6) *safedetails.withSafeDetails (7) *errutil.withMessage (8) *main.withCommandDetails (9) *exec.ExitError

	cluster.go:1516,context.go:135,cluster.go:1505,test_runner.go:829: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2072412-1594189844-87-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		2: 3891
		3: 3827
		1: dead
		8: skipped
		12: skipped
		5: 4173
		6: 4354
		7: 4550
		11: 4279
		9: 4249
		10: 4288
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1789
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1373
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072412&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #50698 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50518 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006220937_v19.2.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006220937_v19.2.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50142 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006091546_v20.1.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006091546_v20.1.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46343 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2130445&tab=buildLog) on [master@aac82be32b138c44bd42200af0c83f8252d594ef](https://github.com/cockroachdb/cockroach/commits/aac82be32b138c44bd42200af0c83f8252d594ef):

```
		  | I200730 13:57:27.965140 84 ccl/workloadccl/fixture.go:586  loaded 21 GiB table history in 42m53.893380894s (150000000 rows, 300000000 index entries, 8.5 MiB)
		  | W200730 15:03:42.275982 1 ccl/workloadccl/fixture.go:497  error enabling automatic stats: dial tcp 10.142.0.74:26257: connect: connection refused
		  | Error: restoring fixture: dial tcp 10.142.0.74:26257: connect: connection refused
		  | Error: COMMAND_PROBLEM: exit status 1
		  | (1) COMMAND_PROBLEM
		  | Wraps: (2) Node 4. Command with error:
		  |   | ```
		  |   | ./workload fixtures load tpcc --warehouses=5000 --scatter --checks=false --partitions=3 --zones="us-east1-b,us-west1-b,europe-west2-b" {pgurl:1}
		  |   | ```
		  | Wraps: (3) exit status 1
		  | Error types: (1) errors.Cmd (2) *hintdetail.withDetail (3) *exec.ExitError
		  |
		  | stdout:
		Wraps: (9) exit status 20
		Error types: (1) *withstack.withStack (2) *errutil.withMessage (3) *withstack.withStack (4) *errutil.withMessage (5) *withstack.withStack (6) *safedetails.withSafeDetails (7) *errutil.withMessage (8) *main.withCommandDetails (9) *exec.ExitError

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2130445-1596093031-78-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		2: 4657
		8: skipped
		3: 4690
		1: dead
		12: skipped
		5: 4215
		6: 4221
		7: 4113
		11: 4202
		10: 4247
		9: 4269
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1373
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=2130445&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #51160 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007071743_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007071743_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50698 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50518 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006220937_v19.2.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006220937_v19.2.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50142 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006091546_v20.1.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006091546_v20.1.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46343 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

At least the last failure is due to a node crash:
```
panic: overlapping descriptors with same gen but different IDs: r11431:/Table/59/1/4{674/7/1231-867/1/2733} [(n1,s1):1, (n9,s9):2, (n6,s6):3, next=4, gen=237, sticky=1596118476.443279478,0] vs r11471:/Table/{59/1/4771/5/1983-60} [(n1,s1):1, (n9,s9):2, (n6,s6):3, next=4, gen=237, sticky=1596118476.443279478,0]
```
Hopefully that'll be fixed in #51395.

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2164877&tab=buildLog) on [master@fdf54ebc69c7d975f8f1dfbf28ced79ddeb83ee4](https://github.com/cockroachdb/cockroach/commits/fdf54ebc69c7d975f8f1dfbf28ced79ddeb83ee4):

```
		  | I200810 13:19:34.327005 90 ccl/workloadccl/fixture.go:586  loaded 21 GiB table history in 1h3m9.55130862s (150000000 rows, 300000000 index entries, 5.8 MiB)
		  | W200810 14:19:04.728059 1 ccl/workloadccl/fixture.go:497  error enabling automatic stats: dial tcp 10.142.0.25:26257: connect: connection refused
		  | Error: restoring fixture: dial tcp 10.142.0.25:26257: connect: connection refused
		  | Error: COMMAND_PROBLEM: exit status 1
		  | (1) COMMAND_PROBLEM
		  | Wraps: (2) Node 4. Command with error:
		  |   | ```
		  |   | ./workload fixtures load tpcc --warehouses=5000 --scatter --checks=false --partitions=3 --zones="us-east1-b,us-west1-b,europe-west2-b" {pgurl:1}
		  |   | ```
		  | Wraps: (3) exit status 1
		  | Error types: (1) errors.Cmd (2) *hintdetail.withDetail (3) *exec.ExitError
		  |
		  | stdout:
		Wraps: (9) exit status 20
		Error types: (1) *withstack.withStack (2) *errutil.withMessage (3) *withstack.withStack (4) *errutil.withMessage (5) *withstack.withStack (6) *safedetails.withSafeDetails (7) *errutil.withMessage (8) *main.withCommandDetails (9) *exec.ExitError

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2164877-1597040336-78-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		1: dead
		8: skipped
		3: 4768
		2: 4577
		12: skipped
		7: 4532
		5: 4271
		6: 4422
		10: 4095
		9: 4168
		11: 4211
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=2164877&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #52337 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008031850_v20.2.0-alpha.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008031850_v20.2.0-alpha.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52206 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51160 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007071743_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007071743_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50698 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50518 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006220937_v19.2.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006220937_v19.2.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50142 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006091546_v20.1.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006091546_v20.1.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46343 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

The latest failure was:
```
panic: pebble: flush-to key (/Table/61/1/2858/63640/0/1558477210.847131843,0) < flushed key (/Table/61/1/3340/90181/0,0)

goroutine 1360102 [running]:
panic(0x47bdf80, 0xc0019c2ca0)
	/usr/local/go/src/runtime/panic.go:722 +0x2c2 fp=0xc0066790d8 sp=0xc006679048 pc=0x7162b2
github.com/cockroachdb/pebble/internal/rangedel.(*Fragmenter).FlushTo(0xc000c5c0f0, 0xc007332a40, 0x13, 0x20)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/internal/rangedel/fragmenter.go:286 +0x828 fp=0xc006679220 sp=0xc0066790d8 pc=0x1618e78
github.com/cockroachdb/pebble.(*compactionIter).Tombstones(0xc006e79400, 0xc007332a40, 0x13, 0x20, 0x0, 0xc007332a40, 0x0, 0x20)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction_iter.go:603 +0xff fp=0xc006679270 sp=0xc006679220 pc=0x166049f
github.com/cockroachdb/pebble.(*DB).runCompaction.func4(0xc00c328580, 0x13, 0x20, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:1898 +0x11c fp=0xc006679480 sp=0xc006679270 pc=0x16a35bc
github.com/cockroachdb/pebble.(*DB).runCompaction(0xc000a87500, 0x411f, 0xc000c5c000, 0x5f5afc0, 0x92e1940, 0xc006827e00, 0xc003163640, 0x6, 0x8, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:2233 +0x15ee fp=0xc006679bc8 sp=0xc006679480 pc=0x165ba9e
github.com/cockroachdb/pebble.(*DB).compact1(0xc000a87500, 0xc000c5c000, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:1669 +0x167 fp=0xc006679e50 sp=0xc006679bc8 pc=0x1659cf7
github.com/cockroachdb/pebble.(*DB).compact.func1(0x5ffa980, 0xc007aa1110)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:1630 +0xbe fp=0xc006679ee0 sp=0xc006679e50 pc=0x16a283e
runtime/pprof.Do(0x5ffa980, 0xc007aa1110, 0xc00000fb60, 0x1, 0x1, 0xc001ed5798)
	/usr/local/go/src/runtime/pprof/runtime.go:37 +0xf8 fp=0xc006679f68 sp=0xc006679ee0 pc=0xf7e428
github.com/cockroachdb/pebble.(*DB).compact(0xc000a87500, 0xc000c5c000, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:1627 +0xa2 fp=0xc006679fc8 sp=0xc006679f68 pc=0x1659b72
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1357 +0x1 fp=0xc006679fd0 sp=0xc006679fc8 pc=0x746a81
created by github.com/cockroachdb/pebble.(*DB).maybeScheduleCompactionPicker
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/pebble/compaction.go:1424 +0x3f4
```

cc. @itsbilal

[(roachtest).tpccbench/nodes=9/cpu=4/multi-region failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2515122&tab=buildLog) on [master@960b4cfc54c8b78df56e62f07d7a07b986ceacff](https://github.com/cockroachdb/cockroach/commits/960b4cfc54c8b78df56e62f07d7a07b986ceacff):

```
		  | I201212 09:54:58.163021 26 ccl/workloadccl/fixture.go:472  imported 641 MiB in new_order table (45000000 rows, 0 index entries, took 1m50.036155629s, 5.82 MiB/s)
		  | W201212 10:24:46.449896 1 ccl/workloadccl/fixture.go:523  error enabling automatic stats: dial tcp 127.0.0.1:26257: connect: connection refused
		  | Error: importing fixture: importing table stock: dial tcp 127.0.0.1:26257: connect: connection refused
		  | Error: COMMAND_PROBLEM: exit status 1
		  | (1) COMMAND_PROBLEM
		  | Wraps: (2) Node 1. Command with error:
		  |   | ```
		  |   | ./cockroach workload fixtures import tpcc --warehouses=5000 --scatter --checks=false --partitions=3 --zones="us-east1-b,us-west1-b,europe-west2-b"
		  |   | ```
		  | Wraps: (3) exit status 1
		  | Error types: (1) errors.Cmd (2) *hintdetail.withDetail (3) *exec.ExitError
		  |
		  | stdout:
		Wraps: (8) exit status 20
		Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *main.withCommandDetails (8) *exec.ExitError

	cluster.go:1637,context.go:140,cluster.go:1626,test_runner.go:841: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2515122-1607756763-43-n12cpu4-geo --oneshot --ignore-empty-nodes: exit status 1 4: skipped
		2: 6210
		3: 5273
		1: dead
		8: skipped
		12: skipped
		7: 5826
		5: 5757
		6: 5725
		11: 5784
		10: 5827
		9: 5685
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | main.glob..func14
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1147
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:271
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1850
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:204
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1374
		Wraps: (3) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *errutil.leafError

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/multi-region](https://teamcity.cockroachdb.com/viewLog.html?buildId=2515122&tab=artifacts#/tpccbench/nodes=9/cpu=4/multi-region)
Related:
- #55939 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #54258 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [T-bulkio](https://api.github.com/repos/cockroachdb/cockroach/labels/T-bulkio) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53540 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed [congmgr assertion failed]  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot)

- #53443 roachtest: tpccbench/nodes=9/cpu=4/multi-region failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fmulti-region.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -613,13 +613,13 @@ func loadTPCCBench(
 	var rebalanceWait time.Duration
 	switch b.LoadConfig {
 	case singleLoadgen:
-		loadArgs = `--scatter --checks=false`
+		loadArgs = `--checks=false`
 		rebalanceWait = time.Duration(b.LoadWarehouses/250) * time.Minute
 	case singlePartitionedLoadgen:
-		loadArgs = fmt.Sprintf(`--scatter --checks=false --partitions=%d`, b.partitions())
+		loadArgs = fmt.Sprintf(`--checks=false --partitions=%d`, b.partitions())
 		rebalanceWait = time.Duration(b.LoadWarehouses/125) * time.Minute
 	case multiLoadgen:
-		loadArgs = fmt.Sprintf(`--scatter --checks=false --partitions=%d --zones="%s"`,
+		loadArgs = fmt.Sprintf(`--checks=false --partitions=%d --zones="%s"`,
 			b.partitions(), strings.Join(b.Distribution.zones(), ","))
 		rebalanceWait = time.Duration(b.LoadWarehouses/50) * time.Minute
 	default:
@@ -646,15 +646,9 @@ func loadTPCCBench(
 	// Split and scatter the tables. Ramp up to the expected load in the desired
 	// distribution. This should allow for load-based rebalancing to help
 	// distribute load. Optionally pass some load configuration-specific flags.
-	method := ""
-	if b.Chaos {
-		// For chaos tests, we don't want to use the default method because it
-		// involves preparing statements on all connections (see #51785).
-		method = "--method=simple"
-	}
 	cmd = fmt.Sprintf("./workload run tpcc --warehouses=%d --workers=%d --max-rate=%d "+
-		"--wait=false --duration=%s --scatter --tolerate-errors %s {pgurl%s}",
-		b.LoadWarehouses, b.LoadWarehouses, b.LoadWarehouses/2, rebalanceWait, method, roachNodes)
+		"--wait=false --duration=%s --scatter --tolerate-errors {pgurl%s}",
+		b.LoadWarehouses, b.LoadWarehouses, b.LoadWarehouses/2, rebalanceWait, roachNodes)
 	if out, err := c.RunWithBuffer(ctx, c.l, loadNode, cmd); err != nil {
 		return errors.Wrapf(err, "failed with output %q", string(out))
 	}
@@ -814,10 +808,9 @@ func runTPCCBench(ctx context.Context, t *test, c *cluster, b tpccBenchSpec) {
 				t.Status(fmt.Sprintf("running benchmark, warehouses=%d", warehouses))
 				histogramsPath := fmt.Sprintf("%s/warehouses=%d/stats.json", perfArtifactsDir, activeWarehouses)
 				cmd := fmt.Sprintf("./workload run tpcc --warehouses=%d --active-warehouses=%d "+
-					"--tolerate-errors --scatter --ramp=%s --duration=%s%s {pgurl%s} "+
-					"--histograms=%s",
+					"--tolerate-errors --ramp=%s --duration=%s%s --histograms=%s {pgurl%s}",
 					b.LoadWarehouses, activeWarehouses, rampDur,
-					loadDur, extraFlags, sqlGateways, histogramsPath)
+					loadDur, extraFlags, histogramsPath, sqlGateways)
 				err := c.RunE(ctx, group.loadNodes, cmd)
 				loadDone <- timeutil.Now()
 				if err != nil {

